Makefile

```diff
@@ -32,6 +32,8 @@ install:
 	go install -tags 'postgres' github.com/golang-migrate/migrate/v4/cmd/migrate@v4.14
 	go install github.com/bojand/ghz/cmd/ghz@v0.105.0
 	go install github.com/gadget-inc/fsdiff/cmd/fsdiff@v0.4
+	go install -v github.com/stamblerre/gocode@latest
```

Comment:
These were required by VSCode golang tooling. Figured I'd add them here

Reply:
Makes sense, can you just drop the `-v` so it lines up with the other installs